{"log":"SERVER ERROR!!! - Error: connect ETIMEDOUT","stack":"Error\n    at Object.__logger.log (F:\\Node\\blingbowling_node\\util\\logger.js:206:12)\n    at shard.singleQuery (F:\\Node\\blingbowling_node\\util\\shard.js:652:12)\n    at shard.singleQuery._async.waterfall.connection.query._logger.log.error (F:\\Node\\blingbowling_node\\util\\shard.js:645:5)\n    at Pool.getConnection (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\Pool.js:48:16)\n    at Handshake.Sequence.end (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\protocol\\sequences\\Sequence.js:75:24)\n    at Protocol.handleNetworkError (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\protocol\\Protocol.js:238:14)\n    at PoolConnection.Connection._handleNetworkError (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\Connection.js:155:18)\n    at Socket.emit (events.js:117:20)\n    at Socket._destroy.self._writableState.errorEmitted (net.js:440:14)\n    at process._tickCallback (node.js:419:13)","level":"error","message":"","timestamp":"2014-11-01T13:03:34.693Z"}
{"log":{"data":"Exception","err":"TypeError: Cannot read property '0' of undefined\n    at User.pDoLogin (F:\\Node\\blingbowling_node\\routes\\p_user.js:77:21)\n    at shard.singleQuery (F:\\Node\\blingbowling_node\\util\\shard.js:653:4)\n    at shard.singleQuery._async.waterfall.connection.query._logger.log.error (F:\\Node\\blingbowling_node\\util\\shard.js:645:5)\n    at Pool.getConnection (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\Pool.js:48:16)\n    at Handshake.Sequence.end (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\protocol\\sequences\\Sequence.js:75:24)\n    at Protocol.handleNetworkError (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\protocol\\Protocol.js:238:14)\n    at PoolConnection.Connection._handleNetworkError (F:\\Node\\blingbowling_node\\node_modules\\mysql\\lib\\Connection.js:155:18)\n    at Socket.emit (events.js:117:20)\n    at Socket._destroy.self._writableState.errorEmitted (net.js:440:14)\n    at process._tickCallback (node.js:419:13)"},"stack":"Error\n    at Object.__logger.log (F:\\Node\\blingbowling_node\\util\\logger.js:206:12)\n    at process.<anonymous> (F:\\Node\\blingbowling_node\\app.js:48:12)\n    at process.emit (events.js:95:17)\n    at process._fatalException (node.js:272:26)","level":"error","message":"","timestamp":"2014-11-01T13:03:39.763Z"}
